### ISSUE TITLE
Editor never loses focus

### ISSUE THREAD

**Issue by varomix** [paulwinex/pw_Houdini_VEX_Editor#5]
Hi, I love the editor
I just have this issue, when I start editing on it, then try to use anything else, the editor keeps focus and I start typing stuff into the editor, it needs to release the focus when you click outside the text area, even if you close the tab the editor still has focus and receives input.

I'm on Linux, Houdini 16.0.676

thank you

**Comment by paulwinex** [comment]
Yes, it is bug in houdini 16.  I will try to fix it but i dont know when. I very busy on different project.

**Comment by varomix** [comment]
ok, thank you
any workarounds in the mean time?

**Comment by varomix** [comment]
any idea when this will be fixed or can you point me in the right direction so I can try to fix it?

thanks

**Comment by paulwinex** [comment]
Nope. I should create new core script. Its take too many time. I dont know when i can((( Sorry.

**Comment by moncifx** [comment]
Hi, I tried it in H17, still have this bug.
